python/eos/plot/plotter.py

```diff
@@ -1970,6 +1972,8 @@ def plot(self):
         if self.output:
             plt.savefig(self.output, bbox_inches='tight', dpi=300)
 
+        return self.fig, self.ax
```

Comment:
OK for now. Will need to revisit if we break API and permit more than one axis per plot.

Reply:
Indeed.